src/ostorlab/agent/kb/kb.py

```diff
@@ -47,6 +48,11 @@ def __getattr__(cls, item: str) -> Entry:
             encoding="utf-8"
         ) as r:
             meta = json.loads(f.read())
+            category_groups = []
+            for category_key, categories_list in meta.get("categories", {}).items():
+                category_group = {"key": category_key, "categories": categories_list}
+                category_groups.append(category_group)
```

Comment:
Could go for list comprehension no ?
categories = meta.get("categories", {})
category_groups = [{'key': k, "categories": v} for k, v in categories.items()]

Reply:
+1.

Reply:
I can do it!